sentry/sentry.conf.example.py

```diff
@@ -306,6 +306,16 @@ def get_internal_network():
 # BITBUCKET_CONSUMER_KEY = 'YOUR_BITBUCKET_CONSUMER_KEY'
 # BITBUCKET_CONSUMER_SECRET = 'YOUR_BITBUCKET_CONSUMER_SECRET'
 
+#######################
+# Discord Integration #
+#######################
+
+# Set the feature to be True if you'd like to enable Discord integration.
+# Refer to https://develop.sentry.dev/integrations/discord/
+SENTRY_FEATURES["organizations:integrations-discord"] = False
```

Comment:
Feature flag for discord will be removed shortly, so this won't be needed. I'll circle back here when that's completed

Reply:
This is now done.
https://github.com/getsentry/sentry/pull/59625